src/cript/nodes/primary_nodes/project.py

```diff
@@ -1,9 +1,152 @@
+from dataclasses import dataclass, field, replace
+from typing import List
+
+from cript.nodes.primary_nodes.collection import Collection
+from cript.nodes.primary_nodes.material import Material
 from cript.nodes.primary_nodes.primary_base_node import PrimaryBaseNode
+from cript.nodes.supporting_nodes.group import Group
 
 
 class Project(PrimaryBaseNode):
     """
-    Project node class as described in the data model
+    Project node
     """
 
-    pass
+    @dataclass(frozen=True)
+    class JsonAttributes(PrimaryBaseNode.JsonAttributes):
+        """
+        all Project attributes
+        """
+
+        # TODO is group needed?
```

Comment:
Group is handled by the nesting of nodes?

Reply:
I believe so, as far as I know, am I wrong?
@dylanwal let me know if I am doing anything wrong here.